src/Differ/DiffConsoleFormatter.php

```diff
@@ -79,7 +79,7 @@ function ($string) use ($isDecorated, $lineTemplate) {
                     "#\n\r|\n#",
                     $isDecorated
                         ? OutputFormatter::escape(rtrim($diff))
-                        : $diff
+                        : rtrim($diff)
```

Comment:
This fixes a small inconsistency I noticed while fixing the test: without colored output, there is an extra line with three spaces after each diff. This does not occur with colored output.